src/main/scala/fi/oph/koski/raportit/AikuistenPerusopetuksenOppijamäärätRaportti.scala

```diff
@@ -45,43 +42,40 @@ case class AikuistenPerusopetuksenOppijamäärätRaportti(db: DB, organisaatioSe
   }
 
   private def query(oppilaitosOidit: List[String], päivä: Date)(implicit u: KoskiSession) = {
-    val calendar = new GregorianCalendar
-    calendar.setTime(päivä);
-    val year = calendar.get(Calendar.YEAR)
+    println("tätä kutsutaan")
+    println(oppilaitosOidit.toString)
     sql"""
     select
       r_opiskeluoikeus.oppilaitos_nimi,
       r_koodisto_koodi.nimi,
-      count(*) as oppilaidenMäärä,
-      count(case when aidinkieli != 'fi' and aidinkieli != 'sv' and aidinkieli != 'se' and aidinkieli != 'ri' and aidinkieli != 'vk' then 1 end) as vieraskielisiä,
-      count(case when pidennetty_oppivelvollisuus = true and vaikeasti_vammainen = true then 1 end) as pidennettyOppivelvollisuusJaVaikeastiVammainen,
-      count(case when pidennetty_oppivelvollisuus = true and vaikeasti_vammainen = false and vammainen = true then 1 end) as pidennettyOppivelvollisuusJaMuuKuinVaikeimminVammainen,
-      count(case when pidennetty_oppivelvollisuus = false and vaikeasti_vammainen = true and erityisen_tuen_paatos = true then 1 end) as virheellisestiSiirretytVaikeastiVammaiset,
-      count(case when pidennetty_oppivelvollisuus = false and vaikeasti_vammainen = false and vammainen = true and erityisen_tuen_paatos = true then 1 end) as virheellisestiSiirretytMuutKuinVaikeimminVammaiset,
-      count(case when erityisen_tuen_paatos = true then 1 end) as erityiselläTuella,
-      count(case when majoitusetu = true then 1 end) as majoitusetu,
-      count(case when kuljetusetu = true then 1 end) as kuljetusetu,
-      count(case when sisaoppilaitosmainen_majoitus = true then 1 end) as sisäoppilaitosmainenMajoitus
-      count(*) as koulukotiOppilaita
+      count(case when opintojen_rahoitus = '1' then 1 end) as oppilaidenMääräVOS,
+      count(case when opintojen_rahoitus != '1' then 1 end) as oppilaidenMääräMuuKuinVOS,
+      count(case when opintojen_rahoitus = '1' and oppimaaran_suorittaja = true then 1 end) as oppimääränSuorittajiaVOS,
```

Comment:
`boolean_muuttuja = true` voi kirjottaa lyhyemmin vain `boolean_muuttuja`. Samoin `boolean_muuttuja = false` voi kirjottaa myös `not boolean_muuttuja`.